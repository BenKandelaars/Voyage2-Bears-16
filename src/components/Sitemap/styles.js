import styled from 'styled-components';

export const Container = styled.div`
  border-top: 1px solid #D1D1D1;  
  margin-bottom: 10px;
`;

export const Wrapper = styled.div `
  display: flex;
  padding: 50px 20px 40px 20px;
  width: 100%;
  justify-content: space-between;
  border-bottom: 1px solid #D1D1D1;

  @media (min-width: 1130px) {
    width: 900px;
    margin: auto; 
  }

  article {
    margin-top: 0px;
    margin-right: 10px;
    padding-top: 0px;
    text-align: left;  
  }
`;

export const Lists = styled.article`
  padding-left: 60px;
  flex-basis: 24%;

  @media (max-width: 750px) {
    display: none;
  }

  h4 {
    margin-top: 0px;
    font-size: 0.9rem;  
    color: #4D4A4A;  
  }

  ul {
    list-style-type: none;
    text-align: left;  
    padding-left: 0px;
    font-size: 0.9rem;
    line-height: 1.25rem;
    color: #5d5a5a;
  }

`;

export const RegionalSelection = styled.article`
  flex-grow: 2;
  display: flex;
  flex-direction: column;

  select {
    font-size: 1.1rem;
    color: #5d5a5a;
    padding: 12px;
    margin-bottom: 16px;
    width: 100%;
  }

  @media (max-width: 750px) {
    flex-direction: row;
   
    select:first-child {
      margin-right: 20px;
    }
  }
`;

