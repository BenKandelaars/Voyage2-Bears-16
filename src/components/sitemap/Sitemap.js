import React from 'react';
import {
  Container,
  Wrapper,
  Lists,
  RegionalSelection,
} from './styles';


export default function Sitemap() {
  return (
    <Container>
      <Wrapper>
        <RegionalSelection>
          <select>
            <option value="english">English</option>
          </select>
          <select>
            <option value="GBP">GBP</option>
          </select>
        </RegionalSelection>
        <Lists>
          <h4><b>Airbnb</b></h4>
          <ul>
            <li>About us</li>
            <li>Careers</li>
            <li>Press</li>
            <li>Policies</li>
            <li>Help</li>
            <li>Diversity &amp; Belonging</li>
          </ul>
        </Lists>
        <Lists>
          <h4><b>Discover</b></h4>
          <ul>
            <li>Trust and Safety</li>
            <li>Travel Credit</li>
            <li>Airbnb Citzen</li>
            <li>Business Travel</li>
            <li>Guidebooks</li>
            <li>Airbnb Mag</li>
          </ul>
        </Lists>
        <Lists>
          <h4><b>Hosting</b></h4>
          <ul>
            <li>Why Host</li>
            <li>Hospitality</li>
            <li>Responsible Hosting</li>
            <li>Community Center </li>
          </ul>
        </Lists>
      </Wrapper>
    </Container>
  );
}
