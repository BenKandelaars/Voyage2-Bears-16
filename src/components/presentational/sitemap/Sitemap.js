import React from 'react';
import {} from './styles';

export default function Sitemap() {
  return (
    <div className="sitemap_container">
      <div className="sitemap">
        <article className="sitemap_regional">
          <select>
            <option value="english">English</option>
          </select>
          <select>
            <option value="GBP">GBP</option>
          </select>
        </article>
        <article className="sitemap_lists">
          <h4><b>Airbnb</b></h4>
          <ul>
            <li>About us</li>
            <li>Careers</li>
            <li>Press</li>
            <li>Policies</li>
            <li>Help</li>
            <li>Diversity &amp; Belonging</li>
          </ul>
        </article>
        <article className="sitemap_lists">
          <h4><b>Discover</b></h4>
          <ul>
            <li>Trust and Safety</li>
            <li>Travel Credit</li>
            <li>Airbnb Citzen</li>
            <li>Business Travel</li>
            <li>Guidebooks</li>
            <li>Airbnb Mag</li>
          </ul>
        </article>
        <article className="sitemap_lists">
          <h4><b>Hosting</b></h4>
          <ul>
            <li>Why Host</li>
            <li>Hospitality</li>
            <li>Responsible Hosting</li>
            <li>Community Center </li>
          </ul>
        </article>
      </div>
    </div>
  );
}
