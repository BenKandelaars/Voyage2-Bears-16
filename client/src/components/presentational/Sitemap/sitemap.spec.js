import React from 'react';
import ReactDom from 'react-dom';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Sitemap from './Sitemap';

describe('Sitemap component tests', () => {
  it('Smoke test', () => {
    const div = document.createElement('div');
    ReactDom.render(<Sitemap />, div);
  });

  it('Snapshot as intented', () => {
    const component = shallow(<Sitemap />);
    expect(toJSON(component)).toMatchSnapshot();
  });
});
