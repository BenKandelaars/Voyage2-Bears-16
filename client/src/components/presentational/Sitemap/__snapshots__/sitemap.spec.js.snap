// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sitemap component tests Snapshot as intented 1`] = `
<styled.div>
  <styled.div>
    <styled.article>
      <select>
        <option
          value="english"
        >
          English
        </option>
      </select>
      <select>
        <option
          value="GBP"
        >
          GBP
        </option>
      </select>
    </styled.article>
    <styled.article>
      <h4>
        <b>
          Airbnb
        </b>
      </h4>
      <ul>
        <li>
          About us
        </li>
        <li>
          Careers
        </li>
        <li>
          Press
        </li>
        <li>
          Policies
        </li>
        <li>
          Help
        </li>
        <li>
          Diversity & Belonging
        </li>
      </ul>
    </styled.article>
    <styled.article>
      <h4>
        <b>
          Discover
        </b>
      </h4>
      <ul>
        <li>
          Trust and Safety
        </li>
        <li>
          Travel Credit
        </li>
        <li>
          Airbnb Citzen
        </li>
        <li>
          Business Travel
        </li>
        <li>
          Guidebooks
        </li>
        <li>
          Airbnb Mag
        </li>
      </ul>
    </styled.article>
    <styled.article>
      <h4>
        <b>
          Hosting
        </b>
      </h4>
      <ul>
        <li>
          Why Host
        </li>
        <li>
          Hospitality
        </li>
        <li>
          Responsible Hosting
        </li>
        <li>
          Community Center 
        </li>
      </ul>
    </styled.article>
  </styled.div>
</styled.div>
`;
